scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -1851,7 +1851,11 @@ class Router(formatOps: FormatOps) {
       case FormatToken(_, T.Ident("|"), _) if rightOwner.is[Pat.Alternative] =>
         val noNL = style.newlines.source.ne(Newlines.keep) || newlines == 0
         Seq(Split(Space.orNL(noNL), 0))
-
+      case FormatToken(_, T.Ident("*"), _)
```

Comment:
should we check that the left is also Ident?

Reply:
I don't think it's needed, the `Term.Repeated` condition is much more important here, which must be after Ident.